*** config.h.in.orig	Tue Oct 24 22:37:42 2006
--- config.h.in	Sat Oct 28 13:10:06 2006
***************
*** 10,19 ****
  #undef HAVE_FTRUNCATE
  
  /* Define to 1 if you have the <getopt.h> header file. */
! #undef HAVE_GETOPT_H
  
  /* Define to 1 if you have the `getopt_long' function. */
! #undef HAVE_GETOPT_LONG
  
  /* Define to 1 if you have the <inttypes.h> header file. */
  #undef HAVE_INTTYPES_H
--- 10,19 ----
  #undef HAVE_FTRUNCATE
  
  /* Define to 1 if you have the <getopt.h> header file. */
! #define HAVE_GETOPT_H 1
  
  /* Define to 1 if you have the `getopt_long' function. */
! #define HAVE_GETOPT_LONG 1
  
  /* Define to 1 if you have the <inttypes.h> header file. */
  #undef HAVE_INTTYPES_H
